src/miner_lora.erl

```diff
@@ -823,3 +893,31 @@ retry_with_rx2(HlmPacket0, From, State) ->
     ChannelSelectorFun = fun(_FreqList) -> Freq end,
     HlmPacket1 = HlmPacket0#packet_pb{rx2_window=undefined},
     send_packet(Payload, TS, ChannelSelectorFun, DataRate, Power, true, HlmPacket1, From, State).
+
+-spec maybe_update_reg_data(State :: state()) -> state().
+maybe_update_reg_data(#state{pubkey_bin=Addr} = State) ->
+    case reg_domain_data_for_addr(Addr, State) of
+        {error, Reason} ->
+            %% Despite having a new chain, we cannot get regulatory domain data for this Hotspot,
+            %% just retry after a second
+            lager:error("cannot confirm regulatory domain for miner ~p, reason: ~p", [
+                libp2p_crypto:bin_to_b58(Addr), Reason
+            ]),
+            erlang:send_after(1000, self(), reg_domain_timeout),
```

Comment:
Why such a hot loop?

Reply:
Updated to be aggressive in testing only